torch/_inductor/ir.py

```diff
@@ -6504,13 +6513,16 @@ def has_side_effects(self):
 @dataclasses.dataclass
 class TorchBindObject(IRNode):
     name: str
-    value: torch._C.ScriptObject
+    value: FakeScriptObject
 
     def get_name(self):
         return self.name
 
+    def get_value(self):
+        return self.value
+
     def get_device(self):
-        return None  # is there a device??
+        return torch.device("cpu")
```

Comment:
Will the Torchbind object ever do compute on a device ?  i'm not sure this is correct. We have to be careful about setting the right device because there's logic in the scheduler for. 

a ) setting up appropriate device guards
b) making sure that existing device codegen is emitted depending on current node (that's the `flush`) stuff 

There was the pr that fixed a case we didn't handle correctly initially. https://github.com/pytorch/pytorch/pull/128275